### PR TITLE
Ubuntu2204

### PR DESCRIPTION
**Pull request by dominikl**
Add installation scripts for Ubuntu 22.04, using Ice and Ice Python binding builds from Glencoe Software([zeroc-ice-ubuntu2204-x86_64](https://github.com/glencoesoftware/zeroc-ice-ubuntu2204-x86_64), [zeroc-ice-py-ubuntu2204-x86_64](https://github.com/glencoesoftware/zeroc-ice-py-ubuntu2204-x86_64))

### CODE REVIEW

File: linux/test/ubuntu2204/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Dockerfile for testing the OMERO Linux installation instructions
+# Not intended for production use
+FROM ubuntu:20.04
```
Comment (jburel) [inline linux/test/ubuntu2204/Dockerfile]: Should be ``ubuntu:22.04``

File: linux/test/ubuntu2204/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+MAINTAINER ome-devel@lists.openmicroscopy.org.uk
+
+ARG OMEROVER=latest
+ARG JAVAVER=openjdk11
+ARG ICEVER=ice36
+ARG PGVER=pg12
```
Comment (jburel) [inline linux/test/ubuntu2204/Dockerfile]: pg14 is the default. 
This variable has no impact since there is option to install anything else implemented

Comment (dominikl) [reply linux/test/ubuntu2204/Dockerfile]: Yes, in the specific shell script I just install `postgresql` which for 22.04 is postgres 14. But I set PGVER to 14, just to make it clear.

**Comment by dominikl** [comment]
Build is passing now: https://github.com/dominikl/omero-install/actions/runs/5178251699/jobs/9329457866 .
Centos builds are failing but due to some unrelated selinux script issue.

File: linux/step01_ubuntu2204_ice_deps.sh [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        build-essential \
+        db5.3-util \
+        libbz2-dev \
+        libdb++-dev \
+        libdb-dev \
+        libexpat-dev \
+        libmcpp-dev \
+        libssl-dev \
+        mcpp \
+        zlib1g-dev
```
Comment (sbesson) [inline linux/step01_ubuntu2204_ice_deps.sh]: Unrelated to this PR but I am unsure all of these packages are required at runtime.

Comment (dominikl) [reply linux/step01_ubuntu2204_ice_deps.sh]: True, installing dev packages is strange. I'll remove them and see what happens.

Comment (jburel) [reply linux/step01_ubuntu2204_ice_deps.sh]: For some dependencies it must be legacy when we were building from source in omero-install

**Comment by dominikl** [comment]
Indeed, the dev packages weren't necessary.

**Pull request merged**